src/libraries/System.Private.CoreLib/src/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -22,6 +23,9 @@ public class DefaultValueAttribute : Attribute
         // Delegate ad hoc created 'TypeDescriptor.ConvertFromInvariantString' reflection object cache
         private static object? s_convertFromInvariantString;
 
+        [FeatureSwitchDefinition("System.ComponentModel.DefaultValueAttribute.IsSupported")]
+        internal static bool IsSupported => AppContext.TryGetSwitch("System.ComponentModel.DefaultValueAttribute.IsSupported", out bool isSupported) ? isSupported : true;
```

Comment:
If this is made `readonly` then the JIT should be able to remove code as well (for cases when the trimmer isn't used).

@sbomer is my `readonly` comment above still applicable? Do we have guidance for this?

Reply:
Hmm, in addition to perhaps supporting JIT, a `static readonly` field accessed by the property would cache the value, so `AppContext.TryGetSwitch()` doesn't get called every time (again, only for non-trimmed scenarios).